File: db4on/reference/vb/db4o-reference-chapters/src/com/db4odoc/f1/enums/EnumExample.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System
Imports com.db4o

Namespace com.db4odoc.f1.Enums
    Class EnumExample
        Public Shared Function FindOpenDoors(ByVal container As ObjectContainer) As ObjectSet
            Dim theDoor As Door = New Door(DoorState.Open)
            Return container.Get(theDoor)
        End Function
    End Class
End Namespace

File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/enums/EnumExample.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */

using System;
using com.db4o;

namespace com.db4odoc.f1.enums
{
	class EnumExample 
	{
		public static ObjectSet FindOpenDoors(ObjectContainer container) 
		{
			return container.Get(new Door(DoorState.Open));
		}
	}
}
